web/src/app/app.component.ts

```diff
@@ -14,19 +14,53 @@ export class AppComponent implements OnInit {
 
   constructor(
     private router: Router,
-    private afAuth: AngularFireAuth,
-    private zone: NgZone,
+    private userService: UserService,
   ) {
     this.version = environment.version;
   }
 
   ngOnInit() {
-    this.afAuth.auth.onAuthStateChanged((user) => {
-      if (user) {
-        this.zone.run(() => {
-          this.router.navigate(['/dashboard']);
-        });
-      }
-    });
+    this.userService.getUser()
+      .subscribe((user) => {
+        if (user) {
+          const userSocial = this.userService.addRefID(user);
+          this.saveUserSocialDetails(userSocial, user.credentials.providerId.split('.')[0]);
+        }
+      });
+  }
+
+  saveUserSocialDetails(userData, provider: string) {
+    const socialDetails = {};
+    socialDetails[provider] = userData;
+    this.userService.checkForSocialDoc(provider, userData.userId)
+      .subscribe((response) => {
+        if (!response) {
+          console.error('error in checking of existing social doc');
```

Comment:
Probably say something like `Error in verifying if the user is an existing one`

Reply:
Okay @SrijithRad Will change it.